### ISSUE TITLE
Feedback needed: break out Utilization by HH Type?

### ISSUE THREAD

**Issue by kiadso** [COHHIO/COHHIO_HMIS#128]
Would it help to differentiate in the Utilization reporting on Rm/Rme between Adults-Only, Adults-Children, and Children-Only beds vs clients? I feel like all the utilization reporting in Rm/Rme is not that helpful for LSA purposes, which is fine if I'm the only one that cares about that aspect, but if it would help literally anyone else, I may as well do this for next year's LSA/PIT/HIC.

**Comment by kiadso** [comment]
Current Utilization looks at unit and bed utilization rates at the project level, which is great, but it seems, based on the LSA warnings they've built, that we're supposed to be looking at utilization at an even more micro level, by Household Type as well.

Talked with CoC team about adding the Household Type dimension to the analysis so that for a given provider, you can see their AC and AO bed and unit utilization across time (Rm) and at the client detail level (Rme). They aren't needing it a lot but feel that if it will make LSAs easier/better then it would be nice to have this.

I'm ranking this as medium priority because it is not a thing that needs to happen prior to vendor transition but it would be really helpful to have prior to the next LSA.

**Comment by kiadso** [comment]
Make an R Markdown doc about this, not something to put in Rm/Rme
